lib/Fhp/Segment/CAZ/HICAZv1.php

```diff
@@ -57,13 +57,13 @@ public function getCamtDescriptor(): string
         return $this->camtDescriptor;
     }
 
-    public function getGebuchteUmsaetze(): Bin
+    public function getGebuchteUmsaetze(): array
```

Comment:
`/** @returns string[] */`
To help IDEs suggest the right things to users and detect bugs.